pkg/workflow/v1/schema.go

```diff
@@ -0,0 +1,19 @@
+package v1
```

Comment:
Would you consider making this package `pkg/workflow/types/v1` instead  of `pkg/workflow/v1` to align with the same in `pkg/container`?

Reply:
done!